lib/ansible/modules/find.py

```diff
@@ -370,6 +370,8 @@ def contentfilter(fsname, pattern, encoding, read_whole_file=False):
     except LookupError as e:
         raise e
     except UnicodeDecodeError as e:
+        if encoding is None:
+            encoding = locale.getpreferredencoding()
```

Comment:
In Python 3.11+ it's a little more similar to `locale.getpreferredencoding(False)`, but trying to keep it in sync with Python implementation details isn't very practical imo.

```suggestion
            encoding = 'None (default determined by the Python built-in function "open")'
```